packages/widgetbook/lib/src/knobs/bool_knob.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:widgetbook/src/knobs/knobs.dart';
+
+class BoolKnob extends Knob<bool> {
+  BoolKnob({
+    required String label,
+    String? description,
+    required bool value,
+  }) : super(
+          label: label,
+          description: description,
+          value: value,
+        );
+
+  @override
+  Widget build() => BooleanKnobWidget(
+        label: label,
+        description: description,
+        value: value,
+      );
+}
```

Comment:
Try `freezed` again for implementing this. It might work if the private constructor is implemented as documented in our private discord conversation 👍🏻

This would make testing a lot easier.